src/code42cli/cmds/alert_rules.py

```diff
@@ -151,24 +149,14 @@ def handle_row(rule_id, username):
 
 def _add_user(sdk, rule_id, username):
     user_id = get_user_id(sdk, username)
-    rules = _get_rule_metadata(sdk, rule_id)
-    try:
-        if rules:
-            sdk.alerts.rules.add_user(rule_id, user_id)
-    except Py42InternalServerError:
-        _check_if_system_rule(rules)
-        raise
+    _get_rule_metadata(sdk, rule_id)  # Verifies rule exists
```

Comment:
This is actually not needed

Reply:
Oh wait yes they - for cases when the rule _Actaully_ does not exist but is not a system rule
just get 2 error messages when is a system rule

Reply:
Oh wait - but then it would still try to do something to a non-existent rule, soooo going to have to re-work this

Reply:
The error message from the new py42 rule Operation sufficies for both cases imo